node/src/components/block_executor.rs

```diff
@@ -235,13 +235,15 @@ impl BlockExecutor {
                     .iter()
                     .map(|&vid| SlashItem::new(vid))
                     .collect();
+                let era_end_timestamp_millis = state.finalized_block.timestamp().millis();
```

Comment:
cc @afck @Fraser999: please confirm that this is right timestamp to use when calling the step function.

Reply:
Yes, that's right.

Reply:
Cool.